FightPandemics/Extensions/Font+Additions.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Font+Additions.swift
+//  FightPandemic [IOS]
+//
+//  Generated on Zeplin. (9/17/2020).
```

Comment:
do we need the Generated on Zeplin comment?

Reply:
Thanks! Will remove.